### PR TITLE
Catherine - qubitekk update

### PR DESCRIPTION
**Pull request by scasagrande**
No mergerino yet, just opening for visibility

### CODE REVIEW

**Comment by CatherineH** [comment]
I don't know why this failed... pylint looks fine to me.

**Comment by scasagrande** [comment]
https://travis-ci.org/Galvant/InstrumentKit/builds/115423502#L788

**Comment by CatherineH** [comment]
thanks for fixing that linting for me!

File: instruments/instruments/thorlabs/tc200.py [hunk 1]
```diff
@@ -91,7 +91,17 @@ def mode(self, newval):
             raise TypeError("Mode setting must be a `TC200.Mode` value, "
                             "got {} instead.".format(type(newval)))
         out_query = "mode={}".format(newval.name)
-        self.sendcmd(out_query)
+        # there is an issue with the TC200; it responds with a spurious
+        # Command Error on mode=normal. Thus, the sendcmd() method cannot
+        # be used.
+        if newval == TC200.Mode.normal:
+            self.prompt = "Command error"
```
Comment (scasagrande) [inline instruments/instruments/thorlabs/tc200.py]: Fix me then we should be good to go

Comment (CatherineH) [inline instruments/instruments/thorlabs/tc200.py]: what do you mean by "fix"?

**Comment by scasagrande** [comment]
SHIP IT :ship:

**Pull request merged**